Fix active, passive, and storyboard hitsound definitions (#6750)
ppy/osu-wiki: wiki/Beatmapping/Hitsound/en.md

## Active hitsound

> A hitsound is considered *active* if it is on a clickable hitobject edge, for example a circle or slider head. These help the player judge whether they clicked early or late by producing a clear impact sound at the time of clicking. If clicked correctly, the sound should match up with the beats in the song the objects are mapped to.

Reviewer: ```suggestion
A hitsound is considered *active* if it is on a clickable part of a hit object, for example a hit circle or slider head. These help the player judge whether they clicked early or late by producing a clear impactful sound at the time of clicking. If played correctly, the sound should match up with a beat in the song.
```
first change is more important i think, others are w/e just my preference

## Passive hitsound

> If a hitsound is on a non-clickable edge, for example a slider repeat or spinner tail, it is considered *passive*. These always play at the same time as long as the player is interacting with the hitobject, unlike active hitsounds. Passive hitsounds neither need to have a clear impact nor be audible, but often are when over beats in the song to help the player maintain rhythm.

Reviewer: ```suggestion
If a hitsound is on a non-clickable part of a hit object, for example a slider repeat or spinner tail, it is considered *passive*. These always play at the same time as long as the player is interacting with the hit object, unlike active hitsounds. Passive hitsounds neither need to have a clear impact nor be audible, but often are when over beats in the song to help the player maintain rhythm.
```
also I've never heard people say "spinner tail". if im not just out of touch id pick a more common example, or reword that one

Reviewer 2: spinner end would be good

Author: It is commonly used throughout the Ranking Criteria and is synonymous with "end", but I do think you have a point because its usage is a bit sporadic. Made [another PR to address that](https://github.com/ppy/osu-wiki/pull/6752).

Reviewer: didn't expect my comment about "spinner tail" to turn into another PR, imo language about sliders is not related & doesn't matter cuz in different contexts people say different things

- "head" usually preferred over "start" for gameplay
- "end" *always* used for gameplay (you drop slider "ends", not "tails")
- "start"/"end" used for parts of skinning
- "start" relatively uncommon in mapping but "head" "end" "tail" all used

and ofc that^ changes over time and with different groups of people. just my experience

but across everyone I've talked to on osu I'm pretty sure I've never heard "spinner tail" which is why i think it should be "spinner end" or something. doesn't have to be consistent with slider language cuz they arent the same.

anyway, it's very minor detail, if u think it's fine as-is then w/e

> If a hitsound is on a non-clickable edge, for example a slider repeat or spinner tail, it is considered *passive*. These always play at the same time as long as the player is interacting with the hitobject, unlike active hitsounds. Passive hitsounds neither need to have a clear impact nor be audible, but often are when over beats in the song to help the player maintain rhythm.

Reviewer 3: This last part reads a little awkwardly

```suggestion
If a hitsound is on a non-clickable edge, for example a slider repeat or spinner tail, it is considered *passive*. These always play at the same time as long as the player is interacting with the hitobject, unlike active hitsounds. Passive hitsounds need neither have a clear impact nor be audible, but often do in order to help the player maintain rhythm.
```

> If a hitsound is on a non-clickable part of a hit object, for example a slider repeat or spinner end, it is considered *passive*. These always play at the same time as long as the player is interacting with the hit object, unlike active hitsounds. Passive hitsounds do not need to have a clear impact or be audible, but often use the same sounds as active hitsounds when over beats in the song in order to help the player maintain rhythm.

Reviewer 3: part of my concern was this wording of "when over beats in the song" that can be expressed more cleanly

```suggestion
If a hitsound is on a non-clickable part of a hit object, for example a slider repeat or spinner end, it is considered *passive*. These always play at the same time as long as the player is interacting with the hit object, unlike active hitsounds. Passive hitsounds do not need to have a clear impact or be audible, but often use the same sounds as active hitsounds when they land on distinct beats in order to help the player maintain rhythm.
```

## Storyboarded hitsound

> Sound samples may be added through storyboarding to mimic hitsounds on objects. Unlike actual hitsounds, however, these always play at the given time regardless of whether the player hit the object. For that reason, they are often not recommended outside of osu!mania.

Reviewer 4: Why is mania a special case here?

Reviewer: can probably split off the part abt mania into a second small paragraph with an explanation of why it's only recommended there (i dont know why myself but assuming it's relevant). otherwise it feels pretty random to include here

Reviewer 3: Storyboarded hitsounds are way more common in mania than osu (from observation), and i think it's to counteract many hitsounds playing in short succession due to players not hitting multiple notes quite simultanously or quick stairs quite in time, etc (can sound bad on fast maps even if you're holding an SS)

Reviewer 4: Yeah I'm not sure. In mania you could just attach hitsounds to one column, that's usually the standard way. I'd probably drop the mania part.

Author: Mainly based this off of

![image](https://user-images.githubusercontent.com/30292137/149966685-d4ee73a4-0605-4116-9d65-819e01c23944.png)

but after asking a few mappers/modders from mania, it turns out they more or less just use them as a last resort when regular hitsounding is either not possible (e.g. to fill the gap between notes) or would require more than one object (e.g. for complex hitsounding or keysounding where multiple samples are used); probably for the reason Walavouchey described.

Either way, dropped the mania part.
